core/README.md

```diff
@@ -0,0 +1,48 @@
+Summary
+------
+
+Solidus Core provides the essential e-commerce data models upon which the
+Solidus system depends.
+
+Core Models
+-----------
+Solidus implements over 200 [models](https://github.com/solidusio/solidus/tree/master/core/app/models/spree),
+and thus a deep inspection of each in this README would be overkill. Instead,
+let's take a quick look at the fundamental models upon which all else depend.
+Currently, these models remain in the Spree namespace as part of the legacy of
+[forking Spree](https://solidus.io/blog/2015/10/28/future-of-spree.html).
+
+* [The Order Sub-System](https://github.com/bbuchalter/solidus/blob/core-readme/core/README.md#the-order-sub-system)
+* [The User Sub-System](https://github.com/bbuchalter/solidus/blob/core-readme/core/README.md#the-user-sub-system)
+* [The Payment Sub-System](https://github.com/bbuchalter/solidus/blob/core-readme/core/README.md#the-payment-sub-system)
+* [The Inventory Sub-System](https://github.com/bbuchalter/solidus/blob/core-readme/core/README.md#the-inventory-sub-system)
+
```

Comment:
I'm not sure if this list is intended to be complete yet, but I'd suggest adding

- Product (including products, variants, taxonomies, properties, options and prices)
- Promotion